commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/PaperCommandRegistration.java

```diff
@@ -0,0 +1,99 @@
+package dev.jorel.commandapi;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.tree.CommandNode;
+import com.mojang.brigadier.tree.LiteralCommandNode;
+import com.mojang.brigadier.tree.RootCommandNode;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+
+/**
+ * Handles logic for registering commands after Paper build 65, where <a href="https://github.com/PaperMC/Paper/pull/8235">https://github.com/PaperMC/Paper/pull/8235</a>
+ * changed a bunch of the behind-the-scenes logic.
+ */
+public class PaperCommandRegistration<Source> extends CommandRegistrationStrategy<Source> {
+	// References to necessary methods
+	private final Supplier<CommandDispatcher<Source>> getBrigadierDispatcher;
+	private final Predicate<CommandNode<Source>> isBukkitCommand;
+
+	// Store registered commands nodes for eventual reloads
+	private final RootCommandNode<Source> registeredNodes = new RootCommandNode<>();
+
+	public PaperCommandRegistration(
+		Supplier<CommandDispatcher<Source>> getBrigadierDispatcher, Predicate<CommandNode<Source>> isBukkitCommand
+	) {
```

Comment:
This is some me kind of nitpick but can we have this on one line?

Reply:
Fair enough. I think I expected to need to add more parameters (like SpigotCommandRegistration), but just two is a reasonable length.